tile/codegen/aggrinit.cc

```diff
@@ -1,5 +1,7 @@
 // Copyright 2018, Intel Corporation
 
+#include <memory>
+#include <set>
 #include <vector>
```

Comment:
Did we need to include these?

Reply:
cpplint complained about their absence, due to use of set<> and make_shared<>.